addon/mixins/route.js

```diff
@@ -151,7 +159,7 @@ const RouteMixin = Ember.Mixin.create({
       throw new Ember.Error("Ember Infinity: You are using an unsupported version of Ember Data.  Please upgrade to at least 1.13.4 or downgrade to 1.0.0-beta.19.2");
     }
 
-    if (Ember.isEmpty(this.get('store')) || Ember.isEmpty(this.get('store')[this._storeFindMethod])){
+    if (Ember.isEmpty(this._store) || Ember.isEmpty(this._store[this._storeFindMethod])){
       throw new Ember.Error("Ember Infinity: Ember Data store is not available to infinityModel");
```

Comment:
PS - I think I misunderstood your comment. :blush:  You just want to change the error message. I leave that to you. Need to catch some :zzz: before busy week starts!